sites/cheerpx/src/content/docs/10-getting-started/index.mdx

```diff
@@ -23,12 +28,13 @@ CheerpX is also available as an ES6 JavaScript module. If you prefer to use the
 
 ```html
 <script type="module">
-	import * as CheerpX from "https://cxrtnc.leaningtech.com/1.0.0/cx.esm.js";
+	// Insert the version number of CheerpX
+	import * as CheerpX from "https://cxrtnc.leaningtech.com/{VERSION_NUMBER}/cx.esm.js";
```

Comment:
This does not work, as expected
![image](https://github.com/user-attachments/assets/5d87d68b-f498-4026-bdc1-8f24b5380e52)

Please use a local build, it does not make sense for me to review a not yet working solution